app-libs/stf/src/trusted_call.rs

```diff
@@ -273,22 +275,44 @@ where
 				Ok(())
 			},
 
-			TrustedCall::pay_as_bid(who, orders_file) => {
+			TrustedCall::pay_as_bid(who, orders) => {
 				let now = Instant::now();
 
-				let raw_orders = fs::read_to_string(&orders_file).map_err(|e| {
-					StfError::Dispatch(format!("Error reading {}. Error: {:?}", orders_file, e))
-				})?;
-				let orders: Vec<Order> = serde_json::from_str(&raw_orders).map_err(|err| {
-					StfError::Dispatch(format!("Error serializing to JSON: {}", err))
-				})?;
+				let ORDERS_DIR = "./records/orders";
+				let RESULTS_DIR = "./records/market_results";
```

Comment:
I would put these into the [best_energy_helpers](https://github.com/olisystems/BEST-Energy/blob/master/app-libs/stf/src/best_energy_helpers.rs) as we will also need the directories in the trusted getter.

Reply:
done